source/qcommon/steam.c

```diff
@@ -125,6 +118,29 @@ void Steam_GetPersonaName( char *name, size_t namesize )
 	if( !namesize ) {
 		return;
 	}
-	UNIMPLEMENTED_DBGBREAK();
-	name[0] = '\0';
+	STEAMSHIM_getPersonaName();
+	const STEAMSHIM_Event *evt;
+	while( !( evt = STEAMSHIM_pump() ) ) {
```

Comment:
First come, first serve?
You should really check that you are handling a persona name message here.
(Same for GetSteamID below)

Reply:
So if it's not a persona name message, then what? I don't think there's any sort of event loop defined for asynchronous messages and ignoring it seems like a bad idea...

Reply:
https://github.com/CoolElectronics/warfork-qfusion/commit/34717e1cef22479e6f13dd769e5b91dee1f1437e for now